resources/views/components/forum/post.blade.php

```diff
@@ -11,6 +11,26 @@ class="post__datetime"
         >
             {{ $post->created_at?->diffForHumans() }}
         </time>
+
+        @if ($post->updated_at > $post->created_at)
```

Comment:
Let's check if updated_at is null instead.

Reply:
Creating a new post will automatically fill the `updated_at` field too (with the same timestamp as the `created_at` field). Therefor I use `>`.

Reply:
Ah right. Sorry about that.